modules/api/test/LorisApiDicomsTest.php

```diff
@@ -0,0 +1,362 @@
+            ),
+            'NULL'
+        );
+        $this->assertSame(
+            gettype(
+                $DicomArray['DicomTars']['0']['SeriesInfo']['1']['InversionTime']
+            ),
+            'string'
+        );
+        $this->assertSame(
+            gettype(
+                $DicomArray['DicomTars']['0']['SeriesInfo']['0']['SliceThickness']
+            ),
+            'string'
+        );
+        $this->assertSame(
+            gettype(
+                $DicomArray['DicomTars']['0']['SeriesInfo']['0']['Modality']
+            ),
+            'string'
+        );
+        $this->assertSame(
+            gettype(
+                $DicomArray['DicomTars']['0']['SeriesInfo']['0']['SeriesUID']
+            ),
+            'string'
+        );
+
+        $this->assertArrayHasKey('Meta', $DicomArray);
+        $this->assertArrayHasKey('CandID', $DicomArray['Meta']);
+        $this->assertArrayHasKey('Visit', $DicomArray['Meta']);
+
+        $this->assertArrayHasKey(
+            'DicomTars',
+            $DicomArray
+        );
+        $this->assertArrayHasKey(
+            '0',
+            $DicomArray['DicomTars']
+        );
+        $this->assertArrayHasKey(
+            'Tarname',
+            $DicomArray['DicomTars']['0']
+        );
+
+        $this->assertArrayHasKey(
+            'SeriesInfo',
+            $DicomArray['DicomTars']['0']
+        );
+        $this->assertArrayHasKey(
+            '0',
+            $DicomArray['DicomTars']['0']['SeriesInfo']
+        );
+        $this->assertArrayHasKey(
+            'SeriesDescription',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'SeriesDescription',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'SeriesNumber',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'EchoTime',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'RepetitionTime',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'InversionTime',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'SliceThickness',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'Modality',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+        $this->assertArrayHasKey(
+            'SeriesUID',
+            $DicomArray['DicomTars']['0']['SeriesInfo']['0']
+        );
+    }
+
+    /**
+     * Tests the HTTP POST request for the
+     * endpoint /candidates/{candid}/{visit}/dicoms
+     * // TODO HANDLING OF POST HANDLING NOT IMPLEMENTED
```

Comment:
If the endpoint do not exists, remove the test. We will add it when we add the endpoint.